### ISSUE TITLE
E(x)-Hentai: Allow Original Size Download While Still Allowing Resized Browsing

### ISSUE THREAD

**Issue by christantoan** [jobobby04/TachiyomiSY#577]
### Describe your suggested feature

Please add an option here to allow download of original size gallery
![Screenshot_20220410-153513](https://user-images.githubusercontent.com/7300310/162609947-b4c9675c-759d-4033-8ecc-4346577a6831.png) but don't remove the option to still browse/stream the resized gallery to save space/traffic/currencies while browsing.

Thank you!

### Other details

Check that there are enough GPs/Credits in the account before starting to download, otherwise show error.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[1.8.1](https://github.com/jobobby04/tachiyomisy/releases/latest)**.
- [X] I will fill out all of the requested information in this form.
